File: Array/selectionSort.cpp
#include<iostream>
using namespace std;

int main(){
    int n=5;
    int arr[5]={1,3,5,2,4};
    cout<<"Before Swap: "<<endl;
    for(int i = 0; i<=n-1; i++)cout<<arr[i]<<" ";

    for(int i = 0; i<=n-2; i++){
        int min = i;
        for(int j = i; j<=n-1; j++){
            if(arr[j]<arr[min]){
                min = j;
            }
        }
        swap(arr[min],arr[i]);
    }
    cout<<endl;
    cout<<"After Swap: "<<endl;
    for(int i = 0; i<=n-1; i++)cout<<arr[i]<<" ";
}

File: Array/selectionSort.py
arr = [1,4,2,3,5]

print("Before Sort:")
for i in arr: print(i,end='')

for i in range(0, len(arr)):
    min = i
    for j in range(i, len(arr)):
        if(arr[j]<arr[i]):
            min = j
            c = arr[i]
            arr[i] = arr[j]
            arr[j] = c
print()
print("After Sort: ")
for i in arr: print(i,end='')